jupyter/d2l-java/chapter_mlp/mlp.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multilayer Perceptrons\n",
+    "\n",
+    ":label:`sec_mlp`\n",
+    "\n",
+    "\n",
+    "In the previous chapter, we introduced \n",
+    "softmax regression (:numref:`sec_softmax`), \n",
+    "implementing the algorithm from scratch \n",
+    "(:numref:`sec_softmax_scratch`) and in gluon \n",
```

Comment:
DJL

Do we need build from scratch version?

Reply:
I think you mean replacing Gluon with DJL ?